clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -7067,6 +7068,20 @@ void OffloadBundler::ConstructJob(Compilation &C, const JobAction &JA,
       Triples += CurDep->getOffloadingArch();
     }
   }
+  bool IsFPGADepBundle = (TCArgs.hasArg(options::OPT_fintelfpga) &&
+      Output.getType() == types::TY_Object);
```

Comment:
Why is a check for `TY_Object` needed? I guess I'm just missing an `fintelfpga` scenario where the second condition wouldn't be met automatically

Reply:
You can bundle up a variety of different types of outputs which wouldn't have us saving the dependency file.  Like preprocessing or creating dependency information with -M.

Reply:
Got it, thanks.